### ISSUE TITLE
FEXConfig: Block size can't be modified with keyboard

### ISSUE THREAD

**Issue by Sonicadvance1** [FEX-Emu/FEX#4121]
![Image_2024-10-17_15-01-02](https://github.com/user-attachments/assets/11c04ddf-38d2-4e66-ac3b-1a6b5dc45026)

Means making large changes impossible and I've fallen back to editing Config.json directly.
No idea what the problem is. QT garbage?

**Comment by neobrain** [comment]
Works for me using arrow up/down or by entering a number directly.

Is this in the CPU or in the Advanced tab?

Which version of Qt did you compile against?

**Comment by Sonicadvance1** [comment]
> Works for me using arrow up/down or by entering a number directly.
> 
> Is this in the CPU or in the Advanced tab?

This is in the CPU tab
> 
> Which version of Qt did you compile against?

This is against QT6

**Comment by neobrain** [comment]
Does the control react in normal ways at all? I.e. can you change the value using the controls on the right? Does the widget get highlighted when hovering with the mouse over it? Does clicking into the widget make it take focus and highlight it?

**Comment by Sonicadvance1** [comment]
> Does the control react in normal ways at all? I.e. can you change the value using the controls on the right? 

Yes, I can change the number using the up and down controls on the right side of it.

> Does the widget get highlighted when hovering with the mouse over it? 

Doesn't highlight when hovering over it.

> Does clicking into the widget make it take focus and highlight it?

Clicking the widget makes it take focus and highlight it, still doesn't allow keyboard input.

**Comment by neobrain** [comment]
Apparently Breeze (default in KDE environments) is the only QML theme that enables keyboard input for spin box controls **by default**, though arrow keys should still be working for you.

Could you verify that adding `editable: true` to https://github.com/FEX-Emu/FEX/blob/0190e1a00bab50ac1f4e854c00413f32a2fe6186/Source/Tools/FEXConfig/main.qml#L224 fixes the problem on your end?

**Comment by Sonicadvance1** [comment]
Yes, adding `editable: true` to that component allows me to edit it by typing on my keyboard.
